_editorial_policies/01_bylaws.md

```diff
@@ -0,0 +1,149 @@
+	    1. All members on the Editorial Board shall be selected in a fair, impartial, and confidential process without regard to race, nationality, religion, gender or gender identity, sexual orientation, or familial status.
+         1. Managing editors will be selected based on:
+            1. all criteria outlined for associate editors, plus demonstrated excellence serving as an associate editor.
+   1. Appointments Terms
+      1. Appointment for all positions are for three years (from the
+         time of adoption of the bylaws., which may be extended for an
+         unlimited number of terms by a vote of two/thirds of the remaining
+         editorial board.
+
+1. Conflicts of interest
+   1. A 'conflict of interest' is defined as any of:
+      1. being a co-author on a paper together for up to 48 months after publication
+      1. a co-awardee on a grant for up to 48 months after the grant, or that currently have a cooperative pending grant.
+      1. if there exists a postdoctoral or graduate advisor relationship at any time in the past.
+      1. at the same institution, or if there is a potential change of institution in progress that would result in being at the same institution.
+      1. having a business relationship (such as serving as a consultant for or part owner of same company) within 48 months.
+
+   1. No associate editor will assign a reviewer who is known to have a conflict of interest with any author.  The explanation of conflict of interest will be explained to the reviewer in their invitation, and the reviewer will affirm that they do not have a conflict of interest.
+
+   1. If later it is discovered that a reviewer has a conflict of interest with an author during the review process, this review will be discarded, and another review will be sought.
+
+   1. If a conflict of interest in a reviewer is found after acceptance, the associate editor will evaluate whether any bias may have affected the acceptance. If the associate editor feels they may be bias, a committee of three associate editors selected by the Section Lead Editor will investigate whether any bias or fraud was intended, and will follow procedures to determine whether admission was fraudulent as outlined by the bylaws on fraud.
+
+   1. No Section Lead Editor shall assign an Associate Editor if they
+have a conflict of interest with a paper. No Associate Editor may
+manage a paper if they have a conflict of interest, and shall notify
+the Section Lead Editor if they do have a conflict.
+
+   1. If a Section Lead Editor has a conflict of interest, the
+assignments to Associate Editors will be made by another Section Lead
+Editor.  The order will be cyclic; if Lead Section Editor A has a
+conflict, it will got to Lead Section Editor B, unless they also have
+a conflict, in which case it will go to C, and so on, returning to A
+when the sections are exhausted. In the case that all of the lead
```

Comment:
I have proposed a narrower definition above.  But we do need to avoid COIs.  We can't be seen as a community publishing our own papers - printing academic money, so to speak!